gruf.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'pry'
 
-  spec.add_runtime_dependency 'grpc', '~> 1.4'
-  spec.add_runtime_dependency 'grpc-tools', '~> 1.4'
+  spec.add_runtime_dependency 'grpc', '~> 1.9'
```

Comment:
Question here - do we _need_ to bump the minimum dependency to 1.9? Can this work with 1.4?

I'd hate to force a downstream upgrade for users if there's not an explicit reason to do so.

Reply:
Honestly probably not. Let me check if updated signal handling code works with 1.4, 1.5 and 1.7

Reply:
Seems to be working fine with 1.4. I can revert.

Reply:
On the other hand `1.4` is 9 months old and there has been a ton of major improvement/bug fixes in grpc ruby code (memory leak fixes, segfault fixes and what not) so I think its a good idea to encourage people to update.

Reply:
Understood, but I don't want to discourage usage because of an unnecessary version requirement. I think it's probably best to leave at 1.4 until we have need to bump it higher. Because it's loosely tied, people are free to push it higher downstream.

Reply:
Additionally `1.9.0` adds ruby 2.5 support.